### ISSUE TITLE
Load Combination Bending Moment Diagram

### ISSUE THREAD

**Issue by egyangelo** [ritchie46/anaStruct#121]
I noticed that bending moment diagram for load combination is not presented accurate and shown as linear value, while it should be 2nd degree curve, although the values are correct.

Bending Moment for Dead Load
![image](https://user-images.githubusercontent.com/64504599/213884081-87721cd3-d85b-4286-b8a4-58021b27dad5.png)

Bending Moment for Live Load
![image](https://user-images.githubusercontent.com/64504599/213884085-7baccadc-fb56-4d6b-b4b0-671b3df919d0.png)

Bending Moment for LoadCombination (D+L)
![image](https://user-images.githubusercontent.com/64504599/213884093-acde7fc7-c81f-4081-9917-49beb4f61195.png)

**Comment by smith120bh** [comment]
Thanks, @egyangelo ! I'll take a look at this shortly (probably this weekend). Looks like the load combination is adding the nodal bending moments, but not the elements' bending moments.

**Comment by smith120bh** [comment]
@egyangelo : Apologies for this taking a bit longer than planned! It is fixed in the just-released version 1.4.1 of anastruct. Please let me know if you still have any issues or if you find any other problems, or have any other suggestions for anastruct!

I tested that this works now using the following code:
```
system = SystemElements()

system.add_element([[0, 0], [2, 0]])
system.add_element([[2, 0], [4, 2]])
system.add_element([[4, 2], [6, 2]])
system.add_support_hinged([1, 4])

lc_dist = LoadCase("dist")
lc_dist.q_load(q=-1, element_id=[1, 2, 3])
system.apply_load_case(lc_dist)
system.remove_loads()

lc_point = LoadCase("point")
lc_point.point_load(node_id=[2, 3], Fy=-2)

combination = LoadCombination("combo")
combination.add_load_case(lc_dist, factor=2)
combination.add_load_case(lc_point, factor=1)
results = combination.solve(system)
results["combination"].show_bending_moment()
```
